docs/ap_guide.md

```diff
@@ -134,82 +134,9 @@ A list of the types to which each property can be assigned is in section 2.4.
-
-##### **distributex**
-  * This function distributes objects along a horizontal plane. 
-  * Parameter ```spacebtwn``` governs the space between objects
-  * Example of use:
-  * ``` (assign (?x x) (function distributex (objects ?x) (settings (spacebtwn 40)))) ```
-##### **distributey**
-  * This function distributes objects along a vertical plane. 
-  * Parameter ```spacebtwn``` governs the space between objects
-  * Example of use:
-  * ```               (assign (?city y) (function distributey (objects ?city)))```
-##### **distribute_grid_around_point**
-  * This function distributes objects within a grid-like structure 
-  *  Parameter ```spacebtwn``` governs the space between objects
-  * Example of use:
-  * ```  (assign (?p x y) (function distribute_grid_around_point (objects ?p)))```
-##### **distribute_within_objects_vertical**
-  * This function distributes objects vertically within the bounds of another object. For example, cars can be distributed within a city.
-  *   Parameter ```spacebtwn``` governs the space between objects
-  * Example of use:
-  * ```(assign (?r x y) (function distribute_within_objects_vertical (objects ?r ?x)(settings (spacebtw 20))))```
-##### **distribute_within_objects_horizontal**
-  * This function distributes objects horizontally within the bounds of another object. For example, cars can be distributed within a city.
-  * Example of use:
-  * ``` (assign (?obj x) (function distribute_within_objects_horizontal (objects ?obj ?loc)))```
-
-
-#### Other Functions
-##### **calculate_label**
-    * ```calculate_label (objects ?obj1 ?obj2))``` displays a numeric label for obj1, based on the number of obj2 objects with this function applied to them. For example, if there are 3 instances of calculate label for ```(a b) (a c) (a d)```, then object ```a``` will have a label of "3". Useful for displaying how many objects are inside another object
-    * Example of use:
-    * ```                (assign (?obj2 label) (function calculate_label (objects ?obj1 ?obj2)))```
-##### **align_middle**
-    * ```align_middle (objects ?x ?y)``` aligns objects x to the middle of  object y
-    * Example of use:
-    * ``` (assign (?x x) (function align_middle (objects ?x ?y)))```
-##### **apply_smaller**
-    * apply_smaller (objects ?x ?y) sets ?y to be displayed smaller than ?x. 
-    * the parameter ```increase_width``` specifies the extend to which an object is wider
-    * Example of us:
-    * ``` (assign (?x width) (function apply_smaller (objects ?x ?y) (settings (increase_width 6))))```
-
-##### **draw_line**
-  * Draws a line between two objects
-  * Example of use:
-  * ```(action (function draw_line (objects ?x ?y)))```
+### 2.4 Functions and Object Properties
+
+Documentations on functions and object properties are presented at: https://planimation-staging-181bc.web.app/documentation
```

Comment:
I'm keen to use the url planimation.planning.domains to point to https://planimation-staging-181bc.web.app/ 

Therefore this link should be updated to planimation.planning.domains/documentation but this will create confusion with the actual documentation https://planimation.github.io/documentation/

It may be worth changing https://planimation-staging-181bc.web.app/documentation to be https://planimation-staging-181bc.web.app/animation-languange and changing the documentation link
![image](https://github.com/planimation/documentation/assets/861782/34472126-84ee-469e-8da6-9cdc55470429)
to be Animation Language.

Reply:
Let me know if we only need to change DNS of planimation.planning.domains to point to https://planimation-staging-181bc.web.app/ or whether we should use another approach in firebase